admin/monitoring/alerts.md

```diff
@@ -0,0 +1,172 @@
+
+1. Login to your slack on your browser.
+2. Create a slack channel where alerts will be posted. We suggest a dedicated channel for alerts that is private (as alerts can potentially contain personally identifiable information) and has all the hub admins added. You can re-use an existing channel that is dedicated to alerts if you have one. We strongly recommend against sending alerts to a channel that's used for general discussions.
+3. Go to [https://api.slack.com/apps?new_app=1](https://api.slack.com/apps?new_app=1) to create a new "App"
+4. Select "From Scratch" in the popup that shows up.
+5. Enter "Grafana Community Alerts (2i2c Hub)" (or a similarly descriptive name) under `App Name`. This will be the name of the bot that posts the alerts.
+6. Select the slack workspace that should get notifications.
+7. Click "Create App" button. This creates a new slack app and redirects you to the configuration page for the new app.
+8. On the left sidebar, select "Incoming Webhooks" under "Features". This opens a page that allows you to create a new webhook URL.
+9. Toggle the switch from "Off" to "On" position on the top right.
+10. Click the "Add New Webhook" button.
+11. Select the workspace and channel (created in step 2) where alerts should be posted to.
+12. Click the "Allow" button.
+13. Copy the webhook URL.
+
+### 2. Connect Grafana to Slack
+
+1. Log-in to your community's Grafana instance with an admin account.
+2. In the left panel, select "Alerting" -> "Contact points"
+
+    ```{tip}
+    If you don't see this option, it is possible you aren't using an admin
+    account! Grafana admin accounts use a username and password you set via
+    an invite link. Ask another admin in your community who *does* have
+    an admin account [to make you one](https://infrastructure.2i2c.org/sre-guide/support/grafana-account/) or
+    reach out to 2i2c support.
+    ```
+
+3. Click "Create contact point".
+4. Select "Slack" under the "Integration" dropdown.
+5. Enter a descriptive name for this contact point under "Name" (such as `2i2c-hub-alerts`).
+6. Paste the webhook URL you copied in the previous section under "Webhook URL".
+7. Press the "Test" button and validate that you are able to see a message in the appropriate slack channel.
+8. Click "Save Contact Point".
+
+### 3. Create a new dashboard folder
+
+The default JupyterHub dashboards, available under the "JupyterHub Default Dashboards" folder
+in your grafana, is managed automatically via the upstream [jupyterhub grafana dashboards](https://github.com/jupyterhub/grafana-dashboards)
+project. So we can't make manual changes or alerts in there, as updates to the
+upstream dashboards may overwrite them.
+
+So we will make a new folder and create new dashboards and alerts inside it.
+Different communities will have different needs, and can maintain their dashboards
+as they wish! We'll provide some guidance for common alerts and dashboards
+in this page, but customize it as you see fit.
```

Comment:
In this example we didn't actually make a dashboard, is it worth mentioning why we need to create a dashboard folder?